modules/shared/programs/claude/files/skills/plan-with-questions/references/consulting-step.md

```
@@ -203,13 +325,79 @@ rm -rf -- "$CONSULT_DIR"
+3. **disqualifier 표시** — 각 옵션의 사용자 노출 description에 "이 선택이 틀릴 수 있는 조건"을 함께 표기. 자문 출력의 `user_facing.plain_disqualifier`를 그대로 사용 (technical `disqualifiers`는 메인 LLM 내부 기록 전용). 평이 disqualifier 누락 시 D2 fallback 4단계(아래) 적용.
 4. **judgment-first** — 트레이드오프 옵션 제시 직전 별도 질문으로 사용자 기준을 먼저 묻는다 (예: "이 결정에서 가장 중요한 기준은? (a) 구현 속도 (b) 운영 안정성 (c) 되돌리기 용이성 (d) 검증 가능성"). 그 다음 옵션 제시.
 
+   **judgment-first 라운드 라벨 부착 절대 금지**: 기준 선택 라운드는 사용자가 *옵션 보기 전에* 추상 기준을 고르는 단계다. 여기에 `(Recommended)` 라벨이 부착되면 anti-anchoring 효과가 source에서부터 무력화된다. 따라서 judgment-first 라운드는 D4 합의 알고리즘을 **실행하지 않으며**, `user_facing.label`만 사용해 기준을 평이하게 표시한다. 본 금지는 `decision_type`이 `tradeoff`인 결정의 judgment-first 사전 라운드에 무조건 적용된다 (자문 출력의 합의 결과와 무관).
+
 상세 메시지 패턴은 [`output-templates.md`](./output-templates.md#step-4--step-i-4-질문-패턴) 참조.
 
+## D2 backward-compat fallback 4단계 (user_facing 누락 시)
+
+자문 출력에 `user_facing` layer가 누락(또는 부분 누락)된 경우 메인 LLM은 graceful degrade로 다음 4단계를 순서대로 시도한다. 어느 단계에서든 사용자 노출 텍스트가 만들어지면 거기서 멈추고 사용자에게 표시한다. **D2 fallback은 D4 Step 2(schema 검증) fail 후의 텍스트 복구 시도이며, D2 fallback이 텍스트를 만들어내도 라벨은 부착하지 않는다** (D4 Step 2가 이미 fallback B로 격하됐으므로). 본 stage 표는 D2 텍스트 복구 흐름이며, 옆의 D4 fallback 표(아래)와는 별개 시스템이다.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)